client/src/localization/languages/Ru.ts

```diff
@@ -739,11 +739,11 @@ export const comparisons = {
   },
   com_ui_pay_per_call: {
     english: 'All AI conversations in one place. Pay per call and not per month',
-    translated: 'Все AI-разговоры в одном месте. Оплачивайте за вызовы, а не за месяц',
+    translated: 'Все AI-чаты в одном месте. Оплачивайте за запрос, а не за месяц',
```

Comment:
thanks no action on this but updating these bottom values is not necessary. I will remove them soon to avoid future confusion